Mathlib/Topology/Category/CompHausLike/Basic.lean

```diff
@@ -166,7 +181,7 @@ end
 variable (P)
 
 /-- The fully faithful embedding of `CompHausLike P` in `TopCat`. -/
-@[simps!]
+@[simps! map]
```

Comment:
Idem: I dropped the `obj` from the `@[simps]` lemmas because it looks like this will cause annoying `simpNF` issues down the line, even though `compHausLikeToTop_obj` doesn't seem to be used anywhere.